stable/ceph-base/bundle.yaml

```diff
@@ -9,12 +10,13 @@ relations:
 - - ceph-osd:mon
   - ceph-mon:osd
 series: focal
-services:
+applications:
   ceph-mon:
     annotations:
       gui-x: '750'
       gui-y: '500'
-    charm: cs:ceph-mon-61
+    charm: ch:ceph-mon
+    channel: quincy/edge
```

Comment:
`source` should be set to focal-yoga UCA.

Reply:
ack